packages/design-tokens/src/scss/generated/_typography.scss

```
@@ -89,6 +89,36 @@ $body-small-text-regular: (
     ),
 ) !default;
 
+$body-xlarge-text-bold: (
+    mobile: (
+        font-family: "'Inter', sans-serif",
+        font-size: 1.25rem,
+        font-style: normal,
+        font-weight: 700,
+        line-height: 1.6,
+    ),
+) !default;
+
+$body-xlarge-text-italic: (
+    mobile: (
+        font-family: "'Inter', sans-serif",
+        font-size: 1.25rem,
+        font-style: normal,
```

I'd expect `italic` here:

```suggestion
        font-style: italic,
```

---

done manually